makefile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+YARPC=yarpc
+TEST_DIR=tests
+
+.PHONY: createVirtualenv
+createVirtualenv:
+	rm -rf env
+	python3 -m pip install --user virtualenv
+	python3 -m venv env
+	source env/bin/activate
+
+.PHONY: preInstall
+preInstall:
+	pip install --upgrade pip
+	pip install -r $(TEST_DIR)/utils/requirements.txt
+	pip install -U pytest
+	pytest --version
+
+.PHONY: lint
+lint:
+
+.PHONY: test
+test: .cleanCoverage
+	pytest $(TEST_DIR)/unit
+
+.cleanCoverage:
+	@echo 'cleaning coverage files ...'
+	rm -f .coverage
+	rm -fr htmlcov/
+
+.PHONY: openReport
+openReport: .cleanCoverage
+	coverage run --branch -m unittest discover --pattern '*_test.py' --start-directory $(TEST_DIR)
```

Comment:
I'd prefer to use pytest-cov for coverage, it's much simpler this way. Current travis configuration does this